cli/src/template.pest

```diff
@@ -25,7 +25,7 @@ raw_literal = @{ literal_char+ }
 literal = { "\"" ~ (raw_literal | escape)* ~ "\"" }
 
 integer_literal = {
-  ASCII_NONZERO_DIGIT ~ ASCII_DIGIT*
+  "-"? ~ ASCII_NONZERO_DIGIT ~ ASCII_DIGIT*
   | "0"
```

Comment:
nit: I think `-0` is also valid.

(We might want a negate operator at some point, but I don't think it's needed right now. Parsing as a literal should be good enough.)